cli/cmd.js

```diff
@@ -102,18 +96,42 @@ const setup = function setup() {
       if (!options.secret ) {return  options.error('secret is required');}
       if (!options.org ) { return  options.error('org is required'); }
       if (!options.env ) { return  options.error('env is required'); }
-      run.start(options,(err)=>{
-        if(cluster.isMaster){
-          if(!err) {
-            console.log("edgemicro started successfully.")
-          } else {
-            console.error(err);
-            process.exit(1);
-          }
-        }
-      });
+      run.start(options);
```

Comment:
Yes. That is true. Moreover, the callback isn't available anymore. Because the we are passing the start file to cluster.setupMaster now.